From: Simon McVittie <smcv@debian.org>
Date: Mon, 18 Dec 2017 11:19:48 +0000
Subject: gdbus-threading test: Allow even longer for
 test_method_calls_in_thread

This should take 4 seconds + overhead, but on a slow or heavily
loaded system, there's no guarantee that it won't take significantly
longer.

Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=884660
Forwarded: no
---
 gio/tests/gdbus-threading.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/gio/tests/gdbus-threading.c b/gio/tests/gdbus-threading.c
index a4ef721..3e4dc92 100644
--- a/gio/tests/gdbus-threading.c
+++ b/gio/tests/gdbus-threading.c
@@ -406,7 +406,7 @@ test_method_calls_on_proxy (GDBusProxy *proxy)
 
       /* elapsed_msec should be 4000 msec +/- change for overhead/inaccuracy */
       g_assert_cmpint (elapsed_msec, >=, 3950);
-      g_assert_cmpint (elapsed_msec,  <, 8000);
+      g_assert_cmpint (elapsed_msec,  <, 30000);
 
       if (g_test_verbose ())
         g_printerr (" ");
